README.md

```diff
@@ -1,2 +1,200 @@
+
+
+<div align
+  <h3><b>README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MATH-MAGICIANS] <a name="about-project"></a>
+
+**[MATH-MAGICIANS]** is a basic project shows how to implement reactjs.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Javascript</a></li>
+     <li><a href="#">HTML</a></li>
+      <li><a href="#">CSS</a></li>
+       <li><a href="#">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Will be added later</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">Will be added later</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]** Configured linters for react app
+- **[key_feature_2]** Basic Setup of React
+- **[key_feature_3]** Professional readme file 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Will be added later](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```  
+  Node
+   VS Code or any other editor
+   
+```
+ 
+
+### Setup
+Clone this repository to your desired folder:
+
+```
+  sh
+  cd my-project
+   git clone https://github.com/sajibarafatsiddiqui/Math-Magicians.git   
+
+```
+
+
+### Install
+
+Install this project with:
+
+```
+   sh
+   cd my-project
+   npm init -y
+   npm install
+   
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+   sh
+   cd math-magicians
+   npm start
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Sajib Arafat Siddiqui**
+
+- GitHub: [@sajibarafatsiddiqui](https://github.com/sajibarafatsiddiqui)
+
+
```

Comment:
**[Optional]**

- [ ] It would be nice if you add your **LinkedIn** and **Twitter** handles here :+1: to make your readme professional :+1: and to show yourself as a branch :blush: